app/src/auto_validator/core/utils/utils.py

```diff
@@ -177,14 +165,16 @@ def install_validator_on_remote_server(
         ssh_manager.copy_files_to_remote(local_coldkey_file, remote_coldkey_path)
 
         # Generate .env file on remote server
-        remote_env_template_path = f"{remote_path}.env.template"
-        remote_pre_config_path = f"{remote_path}pre_config.json"
-        remote_env_path = f"{remote_path}.env"
+        remote_env_template_path = os.path.join(remote_path, ".env.template")
+        remote_pre_config_path = os.path.join(remote_path, "pre_config.json")
+        remote_env_path = os.path.join(remote_path, ".env")
         command = f"python3 {os.path.join(remote_path, 'generate_env.py')} {remote_env_template_path} {remote_pre_config_path} {remote_env_path}"
         ssh_manager.execute_command(command)
+        ssh_manager.logger.info(command)
 
-        # Run pre_install.sh on remote server
-        # ssh_manager.execute_command(f"bash {remote_path}/install.sh")
+        # Run install.sh on remote server
+        remote_install_script_path = os.path.join(remote_path, "install.sh")
+        ssh_manager.execute_command(f"bash {remote_install_script_path}")
```

Comment:
This returns an empty string on error and you don't check it here, so the entire operation will succeed and the user of the web interface won't know about it unless he logs into the server and inspects the logs. Unacceptable. The command should fail and the user should get a message of some sort.